docs/framework/get-started/overview.md

```diff
@@ -93,5 +93,5 @@ The Windows Forms classes are a comprehensive set of reusable types that vastly
 - [Installation guide](../../../docs/framework/install/index.md)
 - [Development Guide](../../../docs/framework/development-guide.md)
 - [Tools](../../../docs/framework/tools/index.md)
-- [.NET Framework Samples](https://msdn.microsoft.com/library/177055f8-4a1f-43e7-aee6-995c196079b1)
+- [.NET Framework Samples](/docs/samples-and-tutorials/index.md)
```

Comment:
```suggestion
- [.NET Framework Samples](../../samples-and-tutorials/index.md)
```

Reply:
Just curious - why don't you allow in-repo links that begin with a slash and end with *ArticleName*.md? The Contributor Guide recommends that to avoid links that begin with a tilde. It seems like a good way to avoid copy-and-paste problems for common links that need to be fixed in different folders/folder levels (like the problem with parent folder counting in one of my previous PRs).

https://github.com/Microsoft/vscode-docs-authoring/issues/135